packages/driver/src/cypress/error_messages.ts

```diff
@@ -985,6 +991,29 @@ export default {
 
         If so, increase \`redirectionLimit\` value in configuration.`
     },
+    cross_origin_load_timed_out ({ ms, configFile, crossOriginUrl, originPolicies }) {
+      return {
+        message: stripIndent`\
+          Timed out after waiting \`${ms}ms\` for your remote page to load on origin(s):
```

Comment:
Are we sure we want the time here without a space? I think I typically see things printed out as:

`3 ms`

rather than

`3ms`

just wanted to make sure this was intentional

Reply:
OK now that I'm looking around I see a mixture so I'm definitely fine either way.

Reply:
We followed the pattern of the other errors messages in this file with timeouts (which use `3ms`).